pkg/pillar/scripts/device-steps.sh

```diff
@@ -319,17 +337,17 @@ access_usb() {
                 echo "$(date -Ins -u) $file not found on $SPECIAL"
             fi
         done
```

Comment:
We still have block `hosts:/config server:/config` on top which will try to copy `hosts` and `server` files from usb to read-only-mounted `/config`. It is not expected.

Reply:
Good catch @giggsoff. Have you seen cases where that is used?
Adding logic to handle a server file will be painful and complex, but the hosts file we can seed via /persist/<somewhere>/hosts.

Reply:
@giggsoff Looks like logic for hosts file is a bit off (which is why I wonder how/if it is used).
If I have a /config/hosts from the build, then it is appended to the overlayfs /etc/hosts on each boot. That's ok.
If I have a hosts file on a USB stick, then it clobbers any /config/hosts. Thus using both the build and the USB stick hosts file wouldn't work as one might expect.

Reply:
I just tried to find places in code where we need write something to config. I have no real cases for this part, probably it will be safe to remove this and add some log if we will hit it occasionally.

Reply:
@giggsoff based on checking around I found that this isn't used as part of installing systems, so I removed this functionality. This means that the override USB stick can only be used to override the DevicePortConfig.